randovania/games/prime3/logic_database/Pirate Homeworld - Research.txt

```diff
@@ -221,18 +221,49 @@ Extra - asset_id: 3594029436312475339
       After Proving Grounds Barrier Removed
   > Event - Gandrayda
       Any of the following:
-          Combat (Expert)
           All of the following:
-              Morph Ball
+              Grapple Lasso and Fight Ghor-G
               Any of the following:
-                  Grapple Lasso and X-Ray Visor
-                  Combat (Beginner) and Enter Hypermode
+                  # Fighting Gandrayda
+                  Combat (Expert)
+                  All of the following:
+                      # Fight with Hypermode
+                      Enter Hypermode
+                      Any of the following:
+                          Combat (Intermediate) or Damage ≥ 399
+                          Combat (Beginner) and Damage ≥ 199
+                  Any of the following:
+                      # Fight Without Hypermode
+                      Damage ≥ 599
+                      Combat (Beginner) and Damage ≥ 299
+                      Combat (Intermediate) and Damage ≥ 199
+                      Combat (Advanced) and Damage ≥ 99
               Any of the following:
-                  Morph Ball Bomb or Boost Ball
-                  Hyper Ball and Knowledge (Beginner) and Enter Hypermode
+                  # Fighting/Avoiding Invisible Phase
```

Comment:
This group reads weird to me. I feel like both combat tricks should be group under invisible objects wince they don't use xray.

Reply:
The idea is that Gandrayda eventually leaves her invisible phase if you wait long enough. Since fighting isn't an option, Combat is used to dodge attacks instead. If it seems like an inappropriate use case, I'll remove the Combat trick option.

Reply:
Ah I see. Then this is fine